lighthouse-core/report/report-generator.js

```diff
@@ -34,6 +34,23 @@ class ReportGenerator {
       return Math.round(totalScore * 100);
     };
 
+    const getItemRating = (value, aggregatorScored) => {
+      if (typeof value === 'boolean') {
+        return value ? 'good' : 'poor';
+      }
+
+      // Limit the rating to average if this is a rating for Best Practices.
+      let rating = aggregatorScored ? 'average' : 'poor';
+      if (value > 0.33) {
```

Comment:
I agree with sam here. If you can improve code immediately involved with what you're changing anyways, you should. Pulling these out would be helpful.